src/main/java/net/openhft/chronicle/queue/impl/single/StoreTailer.java

```diff
@@ -228,16 +236,14 @@ DocumentContext readingDocument0(final boolean includeMetaData) {
             if (tryAgain)
                 next = next0(includeMetaData);
 
-            Wire wire = context.wire();
-            if (wire != null && context.present(next)) {
-                Bytes<?> bytes = wire.bytes();
-                context.setStart(bytes.readPosition() - 4);
-                readingDocumentFound = true;
-                this.lastReadIndex = this.index();
-                return context;
+            // An entry has been found, prepare the context and return it.
+            if (next && (context.wire() != null)) {
```

Comment:
I notice we went from `if (wire != null && context.present(next))` to `if (next && context.wire() == null)` I can see that `context.present()` just returns the value passed in but it has the side-effect :vomiting_face: of setting the `present` field in the context.

Is that change significant?

Reply:
Oh I see it happens in init now